nrf_wifi/fw_if/umac_if/inc/fmac_api_common.h

```diff
@@ -241,6 +241,53 @@ enum nrf_wifi_status nrf_wifi_fmac_get_reg(struct nrf_wifi_fmac_dev_ctx *fmac_de
 enum nrf_wifi_status nrf_wifi_fmac_get_power_save_info(void *fmac_dev_ctx,
 						       unsigned char if_idx);
 
+#ifdef CONFIG_NRF700X_RAWDATA_TX
+/**
+ * @brief Get or set the current mode of operation
+ * @param dev_ctx Pointer to the UMAC IF context for a RPU WLAN device.
+ * @param if_idx Index of the interface on which mode is to be set.
+ * @param mode value to be set for the interface.
+ *
+ * This function is used to send a command
+ *         to RPU to Set/Get mode of operation
+ *
+ * @return Command execution status
+ */
+enum nrf_wifi_status nrf_wifi_fmac_mode(void *dev_ctx,
+					unsigned char if_idx,
+					unsigned char mode);
+
+/**
+ * @brief Get or set the current channel
+ * @param dev_ctx Pointer to the UMAC IF context for a RPU WLAN device.
+ * @param if_idx Index of the interface on which mode is to be set.
+ * @param channel value to be set for the interface.
+ *
+ * This function is used to send a command
+ *         to RPU to Set/Get current channel of operation
+ *
+ * @return Command execution status
```

Comment:
This doesn't look appropriate.

Reply:
The description has been modified and also the packet filter API has been removed to a later PR